contrib/automation_tests/orbit_tracks.py

```diff
@@ -16,10 +16,9 @@ def main(argv):
         ConnectToStadiaInstance(),
         FilterAndSelectFirstProcess(process_filter="hello_ggp"),
         Capture(),
-        # "sdma0" is not present on the DevKits, instead there is "vce0", so this tests for "sdma0 or vce0"
-        VerifyTracksExist(track_names=[
-            "Scheduler", ("*sdma0*", "*vce0*"), "gfx", "All Threads", "hello_ggp_stand"
-        ]),
+        VerifyTracksExist(track_names=["Scheduler", "gfx", "All Threads", "hello_ggp_stand"]),
+        # We check for either "sdma0" or "vce0", to exist. Both are connected to the encoding of video frames.
```

Comment:
```suggestion
        # We check for "sdma0" or "vce0", to exist. Both are connected to the encoding of video frames.
```

Nit: With the allow_duplicates we are actually not looking for "either or" (exclusive).

Reply:
Yes, thanks, fixed.